### PR TITLE
TD-3676 Updates branding and links in the OpenAPI project to NHS England

### PR DESCRIPTION
**Pull request by kevwhitt-hee**
### JIRA link
[TD-3676](https://hee-tis.atlassian.net/browse/TD-3676)

### Description
Updates references to Health Education England in the OpenAPI project to reflect move to NHS England. When we shared the Swagger document (https://uks-learninghubnhsuk-openapi-dev.azurewebsites.net/swagger/index.html) with NHS Staff App colleagues, it was noticed that the branding in the descriptions was still HEE.

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [x] Run the formatter and made sure there are no IDE errors
- [ ] Written appropriate unit tests for the changes, including:
	- accessibility tests for new views
	- tests for new controller methods
	- tests for new or modified API endpoints
- [ ] Manually tested my work with and without JavaScript
- [ ] Tested any Views or partials created or changed with [Wave Chrome plugin](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh/related) and addressed any valid accessibility issues
- [ ] Updated/added documentation in [Confluence](https://hee-tis.atlassian.net/wiki/spaces/TP/pages/3477930003/Learning+Hub) and/or [GitHub Readme](https://github.com/TechnologyEnhancedLearning/LearningHub.Nhs.UserApi/blob/master/README.md). List of documentation links added/changed:
  - [doc_1_here](link_1_here)
- [ ] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing is broken
- [x] Scanned over my pull request in GitHub and addressed any warnings from the GitHub Build and Test checks.

[TD-3676]: https://hee-tis.atlassian.net/browse/TD-3676?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: OpenAPI/LearningHub.Nhs.OpenApi/SwaggerDefinitions/v1.3.0.json [hunk 1]
```diff
@@ -545,12 +545,12 @@
...
-        "type": "apiKey",
-        "name": "X-API-KEY",
-        "in": "header",
-        "description": "Application API key required to authorise all API requests. This must be obtained from the Learning Hub team at Health Education England, Technology Enhanced Learning"
-      },
+        "ApiKey": {
+            "type": "apiKey",
+            "name": "X-API-KEY",
+            "in": "header",
+            "description": "Application API key required to authorise all API requests. This must be obtained from the Learning Hub team at Health Education England, Technology Enhanced Learning, contact us at [england.tel@nhs.net](england.tel@nhs.net)"
```
Comment (ColinBeebyHEE) [inline OpenAPI/LearningHub.Nhs.OpenApi/SwaggerDefinitions/v1.3.0.json:552]: Need to change Health Education England to NHS England on this line.

**Comment by AnjuJose011** [comment]
can see some HEE  references in   services Interfaces and in Helper Class  like BookmarkService.cs . is that will be ok?

**Review by ColinBeebyHEE — APPROVED** [review]

**Review by AnjuJose011 — APPROVED** [review]

**Comment by kevwhitt-hee** [comment]
@AnjuJose011, just to note, the change to StyleCop.json (solution wide) probably means that lots of other copyright statements are now breaking the rule. Should I go through and update all of them in this PR if so?

**Comment by AnjuJose011** [comment]
@kevwhitt-hee    we can do in this PR as it is throwing warnings while building the solution . entirely we can replace  HEE to NHS

**Comment by kevwhitt-hee** [comment]
I've added a commit to address this, @AnjuJose011. @ColinBeebyHEE, starting to have doubts about whether we should have a copyright notice at the top of all of our classes given that we are open-source. Should we be removing them? I'll ask Google too.

**Comment by kevwhitt-hee** [comment]
> I've added a commit to address this, @AnjuJose011. @ColinBeebyHEE, starting to have doubts about whether we should have a copyright notice at the top of all of our classes given that we are open-source. Should we be removing them? I'll ask Google too.

Google seems to think we shouldn't be copyrighting code in an open source platform. Should we remove all of them? What do you both think?

**Review by AnjuJose011 — APPROVED** [review]

**Pull request merged**